spec/play_store/subscription_purchases/subscription_purchase_spec.rb

```diff
@@ -109,6 +109,61 @@
     end
   end
 
+  describe "subscription cancelation by user" do
+    describe "when subscription is not canceled" do
+      let(:fake_subscription_purchase) do
+        FakeSubscriptionPurchase.new(
+          kind: "androidpublisher#subscriptionPurchase",
+          start_time_millis: 1459540113244,
+          expiry_time_millis: 1462132088610,
+          auto_renewing: true,
+          developer_payload: "payload that gets stored and returned",
+          payment_state: 1,
+        )
+      end
+
+      it "is not canceled?" do
+        subject.canceled_by_user?.must_be_false
+      end
+
+      it "returns blank user_cancellation_time_millis" do
```

Comment:
@jnbt @SergeyKishenin sorry for my late reply on that. Had made this request already back then, but since I never worked with the GitHub code review feature, I made a mistake and didn't submit it correctly. It was shown in my UI but I realized it wasn't public yet.